gdk-pixbuf/vendor.md

```diff
@@ -795,7 +782,7 @@ Queries a pointer to the pixel data of a pixbuf.
 # Returns
 
 A pointer to the pixbuf's pixel data.
-Please see the section on [image data](image-data) for information
+Please see the section on [image data][image-data] for information
```

Comment:
Wrong link?

Reply:
I don't know actually...

Reply:
Oh it in GdkPixbuf-2.0.gir too, then we can ignore it